### PR TITLE
Add new "triangles" intro

### PR DESCRIPTION
**Pull request by peppy**
Is the new default, but previous intro can also be chosen.

### CODE REVIEW

**Comment by smoogipoo** [comment]
This can't go in until we have linux/macOS ffmpeg binaries in NativeLibs.

**Comment by peppy** [comment]
Correct, that's coming in the next few hours, hopefully.

**Comment by swoolcock** [comment]
@peppy are you adding those?

**Comment by peppy** [comment]
I am, yeah.

File: osu.Game/Screens/Menu/IntroTriangles.cs [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+                    RelativeSizeAxes = Axes.Both,
+                    Clock = new FramedClock(track),
+                    LoadMenu = LoadMenu
+                }, t =>
+                {
+                    AddInternal(t);
+                    welcome?.Play();
+
+                    // Only start the current track if it is the menu music. A beatmap's track is started when entering the Main Manu.
+                    if (menuMusic.Value)
```
Comment (Game4all) [inline osu.Game/Screens/Menu/IntroTriangles.cs]: This line will cause the game to get stuck on the intro screen if the user has disabled the menu music at startup. Commenting / removing it seems to fix the problem.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**